src/redux/Topic/saga.js

```diff
@@ -35,17 +36,17 @@ export function* calculateWinningsRequestHandler() {
         type: actions.CALCULATE_WINNINGS_RETURN,
         value,
       });
-    } catch (error) {
+    } catch (err) {
       yield put({
         type: actions.CALCULATE_WINNINGS_RETURN,
-        value: { error: error.message ? error.message : '' },
+        error: err.message,
```

Comment:
err.message or err?

Reply:
I am returning just the string currently. If we have any use later to have the entire `error` object, will refactor all the instances later.

I wanted to have a catch-all for all errors in the reducers.